index.bs

```diff
@@ -293,23 +309,38 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
-    account. Note that this includes employing [=user verification=].
+    [=authenticator=]) work in concert to create a [=public key credential=] and associate it with the user's [=[RP]=] account.
+    Note that this typically includes employing a [=test of user presence=] or [=user verification=].
 
 : <dfn>[RP]</dfn>
 :: The entity whose web application utilizes the [=Web Authentication API=] to register and authenticate users. See
     [=Registration=] and [=Authentication=], respectively.
 
-    Note: While the term [=[RP]=] is used in other contexts (e.g., X.509 and OAuth), an entity acting as a [=[RP]=] in one context is
-        not necessarily a [=[RP]=] in other contexts.
+        Note: While the term [=[RP]=] is used in other contexts (e.g., X.509 and OAuth), an entity acting as a [=[RP]=] in one
+        context is not necessarily a [=[RP]=] in other contexts.
 
 : <dfn>Relying Party Identifier</dfn>
 : <dfn>RP ID</dfn>
-:: An identifier for the [=[RP]=] on whose behalf a given registration or authentication ceremony is being performed. Public Key
-    credentials can only be used for authentication by the same entity (as identified by RP ID) that created and registered
-    them. By default, the RP ID for a WebAuthn operation is set to the [=environment settings object/origin=] specified by the
-    [=relevant settings object=] of the {{CredentialsContainer}} object. This default can be overridden by the caller subject
-    to certain restrictions, as specified in [[#createCredential]] and [[#getAssertion]].
+:: A [=domain=] identifier for the [=[RP]=] on whose behalf a given [=registration=] or
+    [=authentication|authentication ceremony=] is being performed. An [=RP ID=] MUST be a [=valid domain string=]. A [=public
+    key credential=] can only be used for [=authentication=] with the same entity (as identified by [=RP ID=]) it was registered
+    with. By default, the [=RP ID=] for a WebAuthn operation is set to the caller's [=environment settings object/origin=]'s
+    [=effective domain=]. This default MAY be overridden by the caller, as long as the caller-specified [=RP ID=] value [=is a
+    registrable domain suffix of or is equal to=] the caller's [=environment settings object/origin=]'s [=effective domain=].
+    See also [[#createCredential]] and [[#getAssertion]].
+
+    <div class="note">
+        Note: [=Public key credential=]'s scope is for a [=[RP]=]'s [=origin=], with the following <em>restrictions</em> and
+        <em>relaxations</em>:
+            - The scheme MUST be `https` (i.e., <em>a restriction</em>), and,
+            - the host MAY be equal to the [=[RP]=]'s [=origin=]'s [=effective domain=], or it MAY be equal to a registrable
+                domain suffix of the [=[RP]=]'s [=origin=]'s [=effective domain=] (i.e., <em>an available relaxation</em>), and,
+            - all (TCP) ports on that host (i.e., <em>a relaxation</em>).
```

Comment:
You have clauses for two bullets, but only a noun phrase for the third, and you ought to have the same grammatical form for all of them. Also, "The scheme must be https, the host may be equal ..., and all ports on that host." doesn't read well.